br/pkg/lightning/backend/local/local.go

```diff
@@ -386,13 +386,13 @@ func checkTiFlashVersion(ctx context.Context, g glue.Glue, checkCtx *backend.Che
 type local struct {
 	engines sync.Map // sync version of map[uuid.UUID]*Engine
 
-	pdCtl     *pdutil.PdController
-	splitCli  split.SplitClient
-	tikvCli   *tikvclient.KVStore
-	tls       *common.TLS
-	pdAddr    string
-	g         glue.Glue
```

Comment:
Glue is to let lightning use its host TiDB's SQLExecutor, will we need it in future? such as lightning write its checkpoint in a table

Reply:
those're not the responsibility of backend, and if it really need we can pass in some object which handles it.